### DISCUSSION TITLE
How do I make searches with CTRL+SHIFT+F be case-insensitive?

### DISCUSSION THREAD

**Discussion by andradei** [wezterm/wezterm#4060] [Q&A] [answered]
The [documentation](https://wezfurlong.org/wezterm/config/lua/keyassignment/Search.html?h=case) shows an example like this:

```lua
  -- search for the string "hash" matching regardless of case
  {
    key = 'H',
    mods = 'SHIFT|CTRL',
    action = act.Search { CaseInSensitiveString = 'hash' },
  },
```

But I'd like to set all searches to be case-insensitive, not with an argument (`'hash'` in the example above).

**Comment by wez** [comment, marked as answer]
try `CaseInSensitiveString = ''` to start with an empty search string, but in case insensitive mode

**Reply by andradei** [reply]
Tried

```lua
local wezterm = require 'wezterm'
return {
  -- ...
  keys = {
    {
      key = 'F',
      mods = 'CTRL|SHIFT',
      action = wezterm.action.Search({ CaseInSensitiveString = '' })
    }
  }
}
```

But no dice.

**Reply by andradei** [reply]
Actually, had to restart Wezterm for some reason to get the change to take effect.
